docs/admin-guides/deployment.md

```diff
@@ -188,6 +188,30 @@ ams:
     local.using-session-catalog-for-hive: true
 ```
 
+### Environments variables
+
+The following environment variables take effect during the startup process of AMS, 
+you can set up those environments to overwrite the default value.
+
+| Environments variable name | Default value      | Description                                | 
+|----------------------------|--------------------|--------------------------------------------|
+| AMORO_CONF_DIR             | ${AMORO_HOME}/conf | location where Amoro loading config files. |
+| AMORO_LOG_DIR              | ${AMORO_HOME}/logs | location where the logs files output       | 
+
+Note: `$AMORO_HOME` can't be overwritten from environment variable. It always points to the parent dir of `ams.sh`.
```

Comment:
The parent dir of `ams.sh` seems to be `bin/`, this seems misleading.

Reply:
fixed